### ISSUE TITLE
Test for message age differences fails

### ISSUE THREAD

**Issue by djwf** [djwf/mail-deduplicate#1]
```
================================================== test session starts ==================================================
platform linux -- Python 3.8.2, pytest-5.4.1, py-1.8.1, pluggy-0.13.1
rootdir: /home/dave/usr/src/maildir-deduplicate.git, inifile: setup.cfg
plugins: cov-2.8.1
collected 7 items / 1 error / 6 selected                                                                                

======================================================== ERRORS =========================================================
____________________________ ERROR collecting maildir_deduplicate/tests/test_deduplicate.py _____________________________
maildir_deduplicate/tests/test_deduplicate.py:287: in <module>
    class TestDateStrategy(TestDeduplicate):
maildir_deduplicate/tests/test_deduplicate.py:292: in TestDateStrategy
    newer_date = newest_date.replace(minutes=-1)
../../local/miniconda3/envs/maildir-dedupe/lib/python3.8/site-packages/arrow/arrow.py:622: in replace
    raise AttributeError('unknown attribute: "{}"'.format(key))
E   AttributeError: unknown attribute: "minutes"

----------- coverage: platform linux, python 3.8.2-final-0 -----------
Name                                 Stmts   Miss Branch BrPart     Cover
-------------------------------------------------------------------------
maildir_deduplicate/__init__.py         43     10      6      0    67.35%
maildir_deduplicate/cli.py              79     45     22      0    33.66%
maildir_deduplicate/deduplicate.py     253    204     98      0    13.96%
maildir_deduplicate/mail.py            126     97     56      0    15.93%
-------------------------------------------------------------------------
TOTAL                                  501    356    182      0    21.23%

================================================ short test summary info ================================================
ERROR maildir_deduplicate/tests/test_deduplicate.py - AttributeError: unknown attribute: "minutes"
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=================================================== 1 error in 0.39s ====================================================
```

**Comment by djwf** [comment]
This is caused by a change in the arrow class - use of replace shift functionality was deprecated, then removed in release 0.14.5 (2019-08-09).

**Comment by djwf** [comment]
Fixed by commit 0f1f2f0
